build.gradle

```diff
@@ -179,6 +180,10 @@ subprojects {
                 "uk.gov.ida:saml-test:${saml_lib_version}",
         )
 
+        proxy_node_shared(
+                "org.bouncycastle:bcpkix-jdk15on:1.62",
```

Comment:
My only concern is whether this is necessary?

Reply:
Thanks. I'll see what happens if I remove it. I remember adding it, but then I also moved the `SelfSignedCertificateGenerator` away from that module.

Reply:
I'm an old fool. I needed to move bouncycastle with the cert generator.